src/components/ambassadorCard/ambassadorCard.module.scss

```
@@ -132,15 +120,51 @@
     &::-webkit-scrollbar-track {
       background: $accent-color;
     }
-  }
-}
 
-.birthday::after {
-  content: url("../../assets/partyHat.png");
-  width: 4rem;
-  height: auto;
-  position: absolute;
-  left: 50%;
-  top: 0;
-  transform: translate(-50%, -90%);
+    h3 {
+      font-size: 0.9rem;
```

```suggestion
      font-size: 0.9rem;
      color: lighten($primary-color, 20%);
```

I feel like adding some color to the headings makes it more readable